src/lib/index.ts

```diff
@@ -12,5 +12,4 @@ export { default as Metadata } from "./Metadata.svelte";
 
 // Actions
 export * from "./utils";
-export { default as tilt } from "./tilt";
-export * from "fluent-svelte"; // a very temporary solution
```

Comment:
This was a very temporary solution indeed @TheOnlyTails haha.

Reply:
Yep, it was just fixed.